VimeoNetworking/Sources/Models/VIMConnection.h

```diff
@@ -71,6 +71,10 @@ extern NSString *const __nonnull VIMConnectionNameNotifications;
 @property (nonatomic, strong, nullable) NSNumber *mainVideosCount;
 @property (nonatomic, strong, nullable) NSNumber *viewableVideosCount;
 
+#pragma mark - Notifications related only
```

Comment:
Can we make a subclass of VIMConnection instead? Seems weird to tack on case-specific properties to this base class (looks like we did this for VOD as well).